### ISSUE TITLE
Right side padding prop

### ISSUE THREAD

**Issue by Gryso** [rajinwonderland/react-code-blocks#24]
Hello and thanks for the library :)

I need to write long ugly one line script and it gets overlaid by copy button. Maybe it would make sense to add option to break code before reaching copy button or moving button outside of container. I solved this using padding in `customStyle` but I find that to be hacky and I think this is legit concern for long lines as rare as they are (should be).

Edge of one line also overflowed outside of container.
![image](https://user-images.githubusercontent.com/47506755/103470900-18f4a580-4d79-11eb-84bf-47062357629f.png)
I do not know what the cause is here as last span does not seem to actually overflow. Here with white background:
![image](https://user-images.githubusercontent.com/47506755/103470944-bcde5100-4d79-11eb-8e3a-50a6e4c6edf1.png)

**Comment by rajinwonderland** [comment]
@Gryso agreed, I'll definitely include text wrapping over padding in the next release. 

Thanks for bringing this up!

**Comment by rajinwonderland** [comment]
@Gryso, had a difficult time trying to reproduce this. Can you help me out by providing me with some context around the bug?
